herbert_core/classes/MPIFramework/messages/MESS_NAMES.m

```diff
@@ -97,10 +114,15 @@
                 mess_codes = num2cell(-1:numel(MESS_NAMES.mess_names_)-2);
                 name_to_code_map_ = containers.Map(MESS_NAMES.mess_names_,mess_codes);
                 code_to_name_map_ = containers.Map(mess_codes,MESS_NAMES.mess_names_);
+                common_tag = name_to_code_map_(MESS_NAMES.persistant_messages_{1});
+                for i=1:numel(MESS_NAMES.persistant_messages_)
+                   pm = MESS_NAMES.persistant_messages_{i};
+                   name_to_code_map_(pm) = common_tag;
+                end
+
             end
             name_to_tag_map = name_to_code_map_;
-            tag_to_name_map = code_to_name_map_;
-        end
+            tag_to_name_map = code_to_name_map_;        end
```

Comment:
end has moved onto the preceding line

Reply:
yes, consequence of merge stash from previous change from tag2name_map to tag_to_name_map and similar. Fixed.